enterprise/j2ee.dd/licenseinfo.xml

```diff
@@ -111,4 +112,29 @@
         <license ref="W3C2" />
         <comment type="CATEGORY_B" />
     </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/bindingschema_3_0.xsd</file>
+        <license ref="EDL-1.0" />
+        <comment type="CATEGORY_B" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/application_9.mdd</file>
```

Comment:
The "mdd" files have a CDDL/GPLv2-CP header. Please don't declare them EPL-2.0 that is plain wrong.

Reply:
@matthiasblaesing I created the mdd file and copy the header from the other mdd files and did not think about the license, Can I change the header to EPL or should I left it as it is?

Reply:
If you created the file yourself from scratch, you have to decide: If the file is based on the XSD (does it derive from it), than I would use the same license as the XSD, if it is independent, please use the ALv2 with the Apache Foundation license assignment. In both cases the license header needs to be adjusted.